src/elements/content-uploader/IconName.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { useIntl } from 'react-intl';
+import { FolderPersonal } from '@box/blueprint-web-assets/icons/Content';
+import { Size8 } from '@box/blueprint-web-assets/tokens/tokens';
+import Badgeable from '../../components/badgeable';
+import FileIcon from '../../icons/file-icon/FileIcon';
+import IconAlertDefault from '../../icons/general/IconAlertDefault';
```

Comment:
addressing this separately. need to get publish a shared feature and update blueprint